crates/bevy_window/src/window.rs

```diff
@@ -881,3 +863,21 @@ pub enum WindowTheme {
     /// Use the dark variant.
     Dark,
 }
+
+impl From<winit::window::Theme> for WindowTheme {
+    fn from(theme: winit::window::Theme) -> Self {
+        match theme {
+            winit::window::Theme::Light => WindowTheme::Light,
+            winit::window::Theme::Dark => WindowTheme::Dark,
+        }
+    }
+}
+
+impl From<WindowTheme> for winit::window::Theme {
+    fn from(theme: WindowTheme) -> Self {
+        match theme {
+            WindowTheme::Light => winit::window::Theme::Light,
+            WindowTheme::Dark => winit::window::Theme::Dark,
+        }
+    }
+}
```

Comment:
Sorry for misleading you here, it's entirely my bad, I didn't understand the implication of what I was recommending.

`winit` should not be a dependency to `bevy_window`. Could you revert to defining the `convert_winit_theme` and `convert_window_theme` functions in `crates/bevy_winit/src/converters.rs`?

Reply:
No problem, updated to reflect the changes here: https://github.com/lewiszlw/bevy/pull/4